content/pt/docs/reference/access-authn-authz/authentication.md

```diff
@@ -0,0 +1,892 @@
+   user: name-of-api-server
+ name: webhook
+```
+ 
+Quando um cliente tenta autenticar-se com o servidor de API utilizando um _bearer token_ como discutido [acima](#adicionando-um-bearer-token-em-uma-requisição), o webhook de autenticação envia um objeto JSON serializado do tipo `TokenReview` contendo o valor do _token_ para o serviço remoto.
+ 
+Note que objetos de API do tipo _webhook_ estão sujeitos às mesmas [regras de compatibilidade de versão](/docs/concepts/overview/kubernetes-api/) como outros objetos de API Kubernetes.
+Implementadores devem verificar o campo de versão da API (`apiVersion`) da requisição para garantir a correta deserialização e **devem** responder com um objeto do tipo `TokenReview` da mesma versão da requisição.
+ 
+{{< tabs name="TokenReview_request" >}}
+{{% tab name="authentication.k8s.io/v1" %}}
+{{< note >}}
+O servidor de API Kubernetes envia por padrão revisão de tokens de versão `authentication.k8s.io/v1beta1` compatibilidade com versões anteriores.
+ 
+Para optar receber revisão de tokens de versão `authentication.k8s.io/v1`, o servidor de API API deve ser inicializado com a opção `--authentication-token-webhook-version=v1`.
+{{< /note >}}
+ 
+```yaml
+{
+ "apiVersion": "authentication.k8s.io/v1",
+ "kind": "TokenReview",
+ "spec": {
+   # Bearer token opaco enviado para o servidor de API
+   "token": "014fbff9a07c...",
+ 
+   # Lista opcional de identificadores de audiência para o servidor ao qual o token foi apresentado
+   # Autenticadores de toke  sensíveis a audiência (por exemplo, autenticadores de token OIDC)
+   # Deve-se verificar que o token foi direcionado a pelo menos um membro da lista de audiência
+   # e retornar a interseção desta lista a audiência valida para o token no estado da resposta
+   # Isto garante com que o token é válido para autenticar-se no servidor ao qual foi apresentado
+   # Se nenhuma audiência for especificada, o token deve ser validado para autenticar-se ao servidor de API do Kubernetes
+   "audiences": ["https://myserver.example.com", "https://myserver.internal.example.com"]
+ }
+}
+```
+{{% /tab %}}
+{{% tab name="authentication.k8s.io/v1beta1" %}}
+```yaml
+{
+ "apiVersion": "authentication.k8s.io/v1beta1",
+ "kind": "TokenReview",
+ "spec": {
+   # Bearer token opaco enviado para o servidor de API
+   "token": "014fbff9a07c...",
+ 
+   # Lista opcional de identificadores de audiência para o servidor ao qual o token foi apresentado
+   # Autenticadores de toke  sensíveis a audiência (por exemplo, autenticadores de token OIDC)
+   # Deve-se verificar que o token foi direcionado a pelo menos um membro da lista de audiência
+   # e retornar a interseção desta lista a audiência valida para o token no estado da resposta
+   # Isto garante com que o token é válido para autenticar-se no servidor ao qual foi apresentado
+   # Se nenhuma audiência for especificada, o token deve ser validado para autenticar-se ao servidor de API do Kubernetes
```

Comment:
same typos as below :) (it seems to be copied from below)

Reply:
Thanks a lot, submitted reviewed content on e8f22d9